src/TimeZones.jl

```diff
@@ -90,12 +64,44 @@ include("local.jl")
 include("ranges.jl")
 include("discovery.jl")
 include("rounding.jl")
-include("parse.jl")
-include("deprecated.jl")
 
 # Required to support Julia `VERSION < v"1.9"`
 if !isdefined(Base, :get_extension)
     include("../ext/TimeZonesRecipesBaseExt.jl")
 end
 
+function update_dates_extension()
+    Dates.CONVERSION_SPECIFIERS['z'] = TimeZone
+    Dates.CONVERSION_SPECIFIERS['Z'] = TimeZone
+    Dates.CONVERSION_DEFAULTS[TimeZone] = ""
+    Dates.CONVERSION_TRANSLATIONS[ZonedDateTime] = (
+        Year, Month, Day, Hour, Minute, Second, Millisecond, TimeZone,
+    )
+end
+
+function __init__()
```

Comment:
I'm pretty sure the `__init__` didn't have to move but I'm fine with it being after the includes